docs/reference/jointparameters.md

```diff
@@ -90,8 +90,9 @@ The `springConstant` field specifies the value of the spring constant (or spring
 The `springConstant` must be positive or zero.
 If the `springConstant` is zero (the default), no spring torque/force will be applied to the joint.
 If the `springConstant` is greater than zero, then a spring force will be computed and applied to the joint in addition to the other forces (i.e., motor force, damping force).
-The spring force is calculated according to Hooke's law: *F = -Kx*, where *K* is the `springConstant` and *x* is the current joint position as represented by the `position` field.
+In case of a linear joint it is expressed in *N/m* and the spring force is calculated according to Hooke's law: *F = -Kx*, where *K* is the `springConstant` and *x* is the current joint position as represented by the `position` field.
 Therefore, the spring force is computed so as to be proportional to the current joint position, and to move the joint back to its initial position.
+Similarly, in case of a rotational joint, the spring constant is expressed in *N.m/rd* and the resulting torque *T* is computed in from this formula: *T = -Kx*.
```

Comment:
```suggestion
Similarly, in case of a rotational joint, the spring constant is expressed in *N.m/rad* and the resulting torque *T* is computed in from this formula: *T = -Kx*.
```

Reply:
According to wikipedia, radians' symbol is rad